rails_api_logger.gemspec

```diff
@@ -29,11 +29,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency "nokogiri"
   spec.add_dependency "zeitwerk", ">= 2.0.0"
 
-  spec.add_development_dependency "sqlite3", "~> 1.4.0"
+  spec.add_development_dependency "sqlite3", ">= 2.1"
```

Comment:
This wants to be > 2, though I didn't check which dependency it was that was dependent on this.